pkg/elle/core/core.go

```diff
@@ -54,22 +79,142 @@ func (c *CycleExplainer) renderCycleExplanation(explainer DataExplainer, circle
 	panic("impl me")
 }
 
-//  A function which takes a history and returns a {:graph, :explainer, :anomalies} map; e.g. realtime-graph.
-type Analyzer func(history History) (Anomalies, DirectedGraph, DataExplainer)
+// Analyzer is a function which takes a history and returns a {:graph, :explainer} map; e.g. realtime-graph.
+type Analyzer func(history History) (DirectedGraph, DataExplainer)
```

Comment:
Perhaps we need retain `Anomalies`, it's used on `Combine` func and rw_register workload.